### PR TITLE
Add issue and pull request templates

### PR DESCRIPTION
**Pull request by denisvieiradev**
# Pull Request

## What I did

Add issue and pull request templates, issue https://github.com/thealphadollar/vanilla-i18n/issues/6

## How I did it
Using [documentation](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/about-issue-and-pull-request-templates) instructions, add files and folders to able use template

## How to verify it
See the changes

## What kind of change does this PR make

- [ ] Major
- [x] Minor
- [ ] Patch

## Description for the changelog

Add issue and pull request templates

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!--
+Customized from the template
+(https://github.com/thealphadollar/vanilla-i18n/blob/master/.github/PULL_REQUEST_TEMPLATE.md)
```
Comment (thealphadollar) [inline .github/PULL_REQUEST_TEMPLATE.md]: I think we should keep the original source of the template here. What do you say?

**Review by thealphadollar — COMMENTED** [review]
Hey Denis, thanks for the contribution. I've suggested a trivial change, can you please see if we can make it?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (denisvieiradev) [reply .github/PULL_REQUEST_TEMPLATE.md]: Nice @thealphadollar , but this path already redirect to correct source ? whats's is wrong ? can you write the correct source to update ?

Comment (thealphadollar) [reply .github/PULL_REQUEST_TEMPLATE.md]: Source should be original, from where you've taken the template. For instance, in the issue template the link was of golang repository which was correct. We are using the template and not the creators, so we should keep the link to the original source (the creators of the template) to give them due credits.

**Review by thealphadollar — CHANGES_REQUESTED** [review]
Please revert the last change made in issue template, we need to keep the link to source that is golang repository.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (denisvieiradev) [reply .github/PULL_REQUEST_TEMPLATE.md]: Nice @thealphadollar !! now I understand . I updated !

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Issue must have
+
+<!--
+Customized from the [template](https://github.com/thealphadollar/vanilla-i18n/blob/master/.github/ISSUE_TEMPLATE.md)
```
Comment (thealphadollar) [inline .github/ISSUE_TEMPLATE.md]: This also needs to be updated I believe, @denisvieira05

**Review by thealphadollar — CHANGES_REQUESTED** [review]
Please revert issue template as well.

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (denisvieiradev) [reply .github/ISSUE_TEMPLATE.md]: done @thealphadollar

**Review by thealphadollar — APPROVED** [review]
LGTM! Thank you @denisvieira05 for your efforts towards the contribution and being patient through the review process. I really enjoyed 🔥

**Comment by denisvieiradev** [comment]
@thealphadollar nice !! you can add hacktoberfest-accepted label on this PR ?

**Comment by thealphadollar** [comment]
@denisvieira05 As far as I'm aware of the rules, all approved and merged PRs in repositories with "hacktoberfest" topic count as contributions :D So, don't worry. Please let me know if this doesn't reflect in your hacktoberfest dashboard.

**Comment by thealphadollar** [comment]
@all-contributors please add @denisvieira05 for maintenance.

**Pull request merged**